### ISSUE TITLE
Openblas license violation

### ISSUE THREAD

**Issue by isuruf** [AnacondaRecipes/r-base-feedstock#1]
Since you are vendoring openblas in this package, please include openblas license in the conda package for r-base as it is mandatory for the BSD license.

**Comment by isuruf** [comment]
cc @msarahan, @mingwandroid, @jjhelmus, @kalefranz

**Comment by msarahan** [comment]
Thanks for pointing this out.  We will certainly rectify this.  We can't retroactively put it in old packages, though.

**Comment by isuruf** [comment]
@mingwandroid, can you look into this please? I find it distateful that Anaconda, Inc would take a month to fix a license violation for an open source project.

**Comment by mingwandroid** [comment]
I agree. Sorry. I'll bring it up with the team today and try to get this scheduled. Apologies.
